R/parallel-simulation-task.R

```diff
@@ -0,0 +1,53 @@
+#' @title ParallelSimulationTask
+#' @description  R6 class for ParallelSimulationTask settings
+ParallelSimulationTask <- R6::R6Class(
+  "ParallelSimulationTask",
+  inherit = SimulationTask,
+  public = list(
+
+    #' @description
+    #' Create a `ParallelSimulationTask` object
+    #' @param ... parameters inherited from R6 class `SimulationTask` object
+    #' @return A new `SimulationTask` object
+    initialize = function(...) {
```

Comment:
hum...why is there  aconstructo that just call base?

Reply:
Just for the documentation, line 9: 'Create a `ParallelSimulationTask` object' instead of 'Create a `SimulationTask` object'.  Remove?

Reply:
I would... esp[ecially because `  #' @return A new `SimulationTask` object` is wrong :)
Constructor is a  standard concept when working with class. Let's remove unless it adds value